guides/v2.2/cloud/live/go-live-checklist.md

```diff
@@ -6,9 +6,7 @@ functional_areas:
   - Testing
 ---
 
-## Pre-Launch Checklist {#checklist}
-
-Prior to launch and switching the DNS, you can download and follow the [Checklist]({{ site.baseurl }}/common/pdf/Pre-Launch.Checklist_4_23_2018.docx) and all tests for your deployed site or store. See an overview of the complete deployment process for Starter and Pro at [Deploy your store]({{ page.baseurl }}/cloud/live/stage-prod-live.html).
+Before you launch to Production, download the [Go live checklist]({{ site.baseurl }}/common/pdf/Pre-Launch.Checklist_4_23_2018.docx) and use it with these instructions to confirm that you have completed all required configuration and testing. See an overview of the complete deployment process for Starter and Pro at [Deploy your store]({{ page.baseurl }}/cloud/live/stage-prod-live.html).
```

Comment:
The phrase "launch to Production" seems a little inaccurate. One would `deploy to the Production environment` and proceed through the launch steps, no?  Especially if it is the first time. May need clarification.